linetools/lists/linelist.py

```diff
@@ -187,9 +187,8 @@ def load_data(self, use_ISM_table=True, tol=1e-3, use_cache=True):
         else:
             # import pdb
             # pdb.set_trace()
-            raise ValueError(
-                'LineList: load_data: Not ready for this LineList name: {:s}'.format(self.list))
-
+            print('LineList: load_data: Not ready for this LineList name: {:s}'.format(self.list))
+            raise ValueError('LineList:  Available options are --  ISM, Strong, EUV, H2, CO, HI, Galaxy')
```

Comment:
It's not removed, just not cached.  Indeed, to access the x-ray lines, you load EUV linelist.